testing_and_setup/compass/ocean/global_ocean/ARM60to9/init/define_base_mesh.py

```diff
@@ -0,0 +1,40 @@
+# /usr/bin/env python
+"""
+% Create cell width array for this mesh on a regular latitude-longitude grid.
+% Outputs:
+%    cellWidth - m x n array, entries are desired cell width in km
+%    lat - latitude, vector of length m, with entries between -90 and 90, degrees
+%    lon - longitude, vector of length n, with entries between -180 and 180, degrees
+"""
+import numpy as np
+import jigsaw_to_MPAS.mesh_definition_tools as mdt
+
+
+def cellWidthVsLatLon():
+    lat = np.arange(-90, 90.01, 0.1)
+    lon = np.arange(-180, 180.01, 0.1)
+
+    QU1 = np.ones(lat.size)
+    EC60to30 = mdt.EC_CellWidthVsLat(lat)
+
+    AtlNH = mdt.mergeCellWidthVsLat(lat, 30 * QU1, 9 * QU1, 10.0, 10.0)
+    AtlGrid = mdt.mergeCellWidthVsLat(lat, EC60to30, AtlNH, 0.0, 1.0)
+
+    PacNH = mdt.mergeCellWidthVsLat(lat, 30 * QU1, 9 * QU1, 45.0, 10.0)
```

Comment:
The high resolution region is controlled by the 9 in the two lines above. The last two arguments are the latitude and width of the transition.